index.js

```diff
@@ -26,6 +27,37 @@ server.listen(port, ipaddress, () => {
 
 const mailbox_storage = JSON.parse(fs.readFileSync("./mailbox.json", "utf8"));
 
+function updateNumbers(){
+	fs.writeFileSync("./json/numbers.json", JSON.stringify(numbers), "utf8");
+	numbers = JSON.stringify(numbers);
+}
+
+function removeNumber(numberIndex){
+	numbers.splice(numberIndex, 1);
+}
+
+schedule.scheduleJob({date: 1, hour: 0, minute: 0, second: 0}, function(){
+	const now = new Date();
+	for (var i in numbers){
+		const number = numbers[i];
+		if (number.year <= now.getFullYear() || number.month <= now.getMonth()){
+			if (number.month == now.getMonth() || (number.month == 12 && now.getMonth() == 0)){
+				// send a notice to the user.
+				var channel = bot.channels.get(number.channel);
+				if (channel != null){ // if the channel is null we will remove them because that means deleted. :(
+					var message = "Your number is expired! Pay your monthly fee by typing `>dial *233`!";
+					channel.send(message);
```

Comment:
u can't send msg if channel is null

also it's undefined not null

Reply:
wait wut ur telling me undefined isn't null. eh i'll just do !channel. i have been doing too much java.